plugins/copr.py

```diff
@@ -256,8 +260,8 @@ def run(self):
 Please do not file bug reports about these packages in Fedora
 Bugzilla. In case of problems, contact the owner of this repository.
 
-Do you really want to enable {0}?""".format('/'.join([self.copr_hostname,
-                                                      copr_username, copr_projectname])))
+""").format('/'.join([self.copr_hostname, copr_username, copr_projectname])))
```

Comment:
You don't need to format this string anymore.

Also the string has two line breaks at the end, and the potential dependencies followup starts with a line break. This results in two empty lines between the two, I think one empty line would be better. I'd also rather not have leading/trailing line breaks in the strings, as it could confuse the translators and they could omit some of them by accident? What about printing the line breaks separately?

Reply:
> You don't need to format this string anymore.

Good catch.

> Also the string has two line breaks at the end, and the potential dependencies
> followup starts with a line break. This results in two empty lines between the
> two, I think one empty line would be better. I'd also rather not have
> leading/trailing line breaks in the strings, as it could confuse the
> translators and they could omit some of them by accident? What about printing
> the line breaks separately?

Yeah, good idea ... done in separate wrapping function now, by calling
msg.strip() first.